### PR TITLE
REDSHOP-5949: Admin > Order detail view > Update discount error

### LINKED ISSUE #5279: [REDSHOP-5949] Admin > Order detail view > Update discount error
If you update the discount on a order and update again with the same amount it triggers COM_REDSHOP_ERROR_UPDATING_DISCOUNT

I don´t know if thats meant to be, but its still stange it triggers a message like this.
The text is a error text, but the message type is "message" and not "error". So you don´t really notice the error because the system-message-container is green and not red.

### PR DESCRIPTION
**Pull request by hoang-tran-redweb**
add style error when display message error

### CODE REVIEW

File: component/admin/controllers/order_detail.php [hunk 1]
```diff
@@ -201,9 +201,10 @@ public function update_discount()
             $msg = JText::_('COM_REDSHOP_DISCOUNT_UPDATED');
```
Comment (anhredweb) [inline component/admin/controllers/order_detail.php:201]: @hoang-tran-redweb : please also set $type = 'success' for of coming `else` condition

File: component/admin/controllers/order_detail.php [hunk 2]
```diff
@@ -218,9 +219,10 @@ public function special_discount()
             $msg = JText::_('COM_REDSHOP_SPECIAL_DISCOUNT_APPLIED');
```
Comment (anhredweb) [inline component/admin/controllers/order_detail.php:219]: @hoang-tran-redweb same here.

**Review by anhredweb — CHANGES_REQUESTED** [review]
@hoang-tran-redweb please careful for variable when using, missing $type definition for success case may lead to warning or notice.

**Review by anhredweb — DISMISSED** [review]

**Comment by VanTrieuThanh** [comment]
@hoang-tran-redweb help me check with Scrutinizer

**Review by anhredweb — APPROVED** [review]

**Pull request merged**